python/tests/Examples.py

```diff
@@ -695,6 +696,94 @@ def test_create_a_cluster_with_link_and_confidence(self):
         self.dump_graph(g, "create a cluster with link and confidence")
 
 
+    def test_create_an_event_with_ldc_time(self):
+        g = aifutils.make_graph()
+        g.bind('ldcOnt', SEEDLING_TYPES_NIST.uri)
+
+        # every AIF needs an object for the system responsible for creating it
+        system = aifutils.make_system_with_uri(g, "http://www.test.edu/testSystem")
+
+        # create a start position event with unknown start and end time
+        event_start_position = aifutils.make_event(g, "http://www.test.edu/event/1", system)
+        aifutils.mark_type(g, "http://www.test.edu/assertions/1", event_start_position, SEEDLING_TYPES_NIST['Personnel.StartPosition'], system, 1.0)
+        unkown = LDCTimeComponent(LDCTimeType.UNKONWN, None, None, None)
+        endBefore = LDCTimeComponent(LDCTimeType.BEFORE, "2016", None, None)
+        aifutils.mark_ldc_time(g, event_start_position, unkown, endBefore, system)
+
+        # create an attack event with an unkown start date, but definite end date
+        event_attack_unknown = aifutils.make_event(g, "http://www.test.edu/event/2", system)
+        aifutils.mark_type(g, "http://www.test.edu/assertions/2", event_attack_unknown, SEEDLING_TYPES_NIST['Conflict.Attack'], system, 1.0)
+        start = LDCTimeComponent(LDCTimeType.AFTER, "2014", "--02", None)
+        end = LDCTimeComponent(LDCTimeType.ON, "2014", "--02", "---21")
+        aifutils.mark_ldc_time(g, event_attack_unknown, start, end, system)
+
```

Comment:
Please use the new_file function to create files when validation script is being used.